rules/no_large_snapshots.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = context => {
+  if (context.getFilename().endsWith('.snap')) {
+    const lineLimit = context.options[0] || 50;
```

Comment:
I think it should take an object as options, just to be easier to expand in the future.

Reply:
Yeah I was debating that. Will do.